scripts/obtain/dotnet-install.sh

```diff
@@ -188,7 +194,7 @@ get_legacy_os_name() {
         fi
     fi
 
-    say_verbose "Distribution specific OS name and version could not be detected: $ID.$VERSION_ID"
```

Comment:
This didn't actually work as $ID and $VERSION_ID are undefined in the error scenario